code/game/objects/items/weapons/material/kitchen.dm

```diff
@@ -51,15 +51,15 @@
 			if (fullness > (550 * (1 + M.overeatduration / 2000)))
 				to_chat(M, "You cannot force anymore food down!")
 				return
-			M.visible_message(SPAN_NOTICE("\The [user] [is_liquid ? "drinks" : "eats"] some [loaded] from \the [src]."))
```

Comment:
I think this should still remain a visible message - just on a cooldown, and/or decrease the visible_message range

Reply:
I was considering it, but ultimately I decided on this, because no other form of eating food (Such as with your hands, or drinking from a cup) produces a visible_message. It feels more uniform this way.